### PR TITLE
get locators only incase of UI upgrade

### PR DESCRIPTION
**Pull request by vavuthu**
Fixes: #5271 

Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Review by petr-balogh — DISMISSED** [review]

File: ocs_ci/ocs/ocs_upgrade.py [hunk 1]
```diff
@@ -644,14 +644,26 @@ def ocs_odf_upgrade_ui():
     logger.info(
```
Comment (am-agrawa) [inline ocs_ci/ocs/ocs_upgrade.py:644]: Please move this logger just before this line so as to maintain the code continuity. 
```
val_obj.do_click(
        upgrade_ocs.validation_loc["storage-system-on-installed-operators"]
    )
```

**Review by jilju — DISMISSED** [review]

File: ocs_ci/ocs/ocs_upgrade.py (hunk 1 above)
Comment (am-agrawa) [inline ocs_ci/ocs/ocs_upgrade.py:644]: Please move this logger just before this line so as to maintain the code continuity.
```
val_obj.do_click(
        upgrade_ocs.validation_loc["storage-system-on-installed-operators"]
    )
```

**Review by am-agrawa — CHANGES_REQUESTED** [review]

**Review by OdedViner — APPROVED** [review]

**Review by am-agrawa — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]

File: ocs_ci/ocs/ocs_upgrade.py (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/ocs/ocs_upgrade.py]: done

**Review by shylesh — APPROVED** [review]
LGTM

**Comment by am-agrawa** [comment]
Merging it on behalf of @vavuthu to unblock 4.10 deployment. 
Thanks @shylesh for approving it.

**Pull request merged**